components/domains/registerCheckboxes.tsx

```
@@ -6,83 +6,86 @@ import { CurrencyType } from "@/utils/constants";
-          onChange={onChangeTermsBox}
-          variant={variant}
-        />
-      </div>
-    </div>
-  );
+	const getHelperText = (): string => {
+		return `Enabling a subscription permits Starknet ID to  domain automatically every year for you! This approval gives us only the possibility to  domain once per year ${
+			maxPriceRange
+				? `(maximum ${weiToEth(maxPriceRange).toFixed(
+						3
+				  )} ${displayedCurrency}/year)`
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)